linux/entrypoint/pre_start.sh

```diff
@@ -12,6 +12,7 @@ $ENTRYPOINT_HOME/certs.sh
 
 props set org.codice.ddf.system.hostname $_app_hostname $APP_HOME/etc/system.properties
 props set org.codice.ddf.system.siteName $_app_hostname $APP_HOME/etc/system.properties
+props set org.codice.ddf.system.httpsPort $HTTPS_PORT $APP_HOME/etc/system.properties
```

Comment:
You'll just need to remove this line now that it is done in side the conditional farther down. After that we should be all set 👍